src/test/java/net/mcreator/integration/generator/GTFeatureBlocks.java

```diff
@@ -44,6 +44,7 @@
 import static org.junit.jupiter.api.Assertions.fail;
 
 public class GTFeatureBlocks {
+	private static final String[] specialCases = {"block_predicate_not", "int_provider_clamped"};

```

Comment:
You can as well define it as a list right here, can't you?

Reply:
As contains is used, possibly a static list here as you suggested is actually better (List.of)